### PR TITLE
fix: Update sharing

### PR DESCRIPTION
**Pull request by y-lohse**
Fixes copy link button

### CODE REVIEW

**Review by Crash-- — APPROVED** [review]

**Comment by ptbrowne** [comment]
AMHA no need to wait for a review to merge this kind of PR. It's nice to have a PR to increase shared knowledge but there is no need to wait to merge it after it passes the tests.

**Comment by y-lohse** [comment]
I agree, I wasn't waiting or requesting a review, but as code owners you get automatically added ;)

**Pull request merged**
